drivers/adodb-mysqli.inc.php

```diff
@@ -120,6 +125,11 @@ function _connect($argHostname = NULL,
 		//http ://php.net/manual/en/mysqli.persistconns.php
 		if ($persist && PHP_VERSION > 5.2 && strncmp($argHostname,'p:',2) != 0) $argHostname = 'p:'.$argHostname;
 
+		// SSL Connections for MySQLI
+		if ($this->ssl_key || $this->ssl_cer || $this->ssl_ca || $this->ssl_ccapath || $this->ca_cipher) {
```

Comment:
ssl_ccapath? typo?

Reply:
100% correct, thank you peterdd. Repaired.